Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -190,17 +187,20 @@ static Color GetAccentColor()
 
 		class AndroidDeviceInfo : DeviceInfo
 		{
-			readonly IDeviceInfoProvider _formsActivity;
+			readonly Context _formsActivity;
 			readonly Size _pixelScreenSize;
 			readonly double _scalingFactor;
 
 			Orientation _previousOrientation = Orientation.Undefined;
 
-			public AndroidDeviceInfo(IDeviceInfoProvider formsActivity)
+			public AndroidDeviceInfo(Context formsActivity)
```

Comment:
What happens if you remove the parameter and just use `Context` everywhere inside this class? I'm not sure what kind of purpose `IDeviceInfoProvider` serves.